src/tests/Common/helixpublishwitharcade.proj

```diff
@@ -723,6 +723,10 @@
   <ItemGroup Condition=" '$(UsesHelixSdk)' == 'true' ">
     <HelixCorrelationPayload Include="$(CoreRootDirectory)" />
     <HelixCorrelationPayload Include="$(XUnitLogCheckerDirectory)" />
+    <HelixCorrelationPayload Condition="'$(TestWrapperTargetsWindows)' == 'true'" Include="dotnet-sos">
+      <Destination>sos</Destination>
+      <Uri>https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/flat2/dotnet-sos/$(DotnetSosVersion)/dotnet-sos.$(DotnetSosVersion).nupkg</Uri>
```

Comment:
Before dotnet-sos existed, we used clrmd directly in RemoteExecutor:
https://github.com/dotnet/arcade/blob/bdc59254cf108e1d48451dc43bb9ebc331cdca7b/src/Microsoft.DotNet.RemoteExecutor/src/RemoteInvokeHandle.cs#L177-L225
We might want to standardize on one or the other.

Reply:
ClrMD there is doing process attach only for timeouts, whereas we're handling both timeouts and crashes here, so it's a little different. Definitely worth considering consolidation though.

Reply:
There were some talks about standardizing this sort of mechanism (dumping stack traces from a dump to stdout for the build analysis tooling), but with the recent changes to the EngSrv teams, I don't know how much of that work will still happen.

Reply:
Right, I understand they're handling different cases.  But they can both handle both, so it's overkill to have two different tools used for the same purpose.  Simply suggesting we choose one and stick with it.

Reply:
I'll look at the RemoteExecutor implementation and see if we can consolidate something useful.

Reply:
Cool, thanks.

Reply:
We'll need to use something like this pattern to get both native and managed stack traces, and we'll likely want to use this model as well to solve https://github.com/dotnet/runtime/issues/83047 (which focuses more on crashes than timeouts) if we don't move to using `dotnet test` for the libraries tests in Helix. The RemoteExecutor version has a much better UX around the output though as it has more structure to work with (from CLRMD's APIs).